userdocs/profiles.dev/docs/installer-docs/removing-profiles.md

```diff
@@ -7,5 +7,5 @@ sidebar_position: 6
 To uninstall any profile from your cluster, just remove the directory from your
 GitOps repo, and open a PR with that commit.
 
-After you approve and merge the PR, Flux will sync the changes and the profile will
+After you approve and merge the PR, Flux will sync the changes, and the profile will
```

Comment:
The comma there feels like an unnatural pause to me 🤔

Reply:
I'll remove it. :)

Reply:
Although there is a thing where if an `and` separates a sentece for which the two parts are independent (i.e. not causality wise, but gramatically), than the `and` requires a comma. :D

Reply:
Like I did here:

> Flux will then sync the new files, and the profile will be applied to your cluster.

Reply:
Yep I saw that, and it fits better there. And yes technically you are correct that here the sentence is 2 clauses so a comma is appropriate. But because here the first clause is longer there is a different rhythm to the whole thing, and a second comma breaks it.

Reply:
But yes syntactically you are correct 🙃

Reply:
![48c](https://user-images.githubusercontent.com/182850/126353608-d2cb4b82-c361-4525-a12b-e53ceb13b4bb.jpg)